### PR TITLE
HW#2

### PR DESCRIPTION
**Pull request by AberaKebede**
3

### CODE REVIEW

File: class03/abera/wine.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#1 add a wine of your choice to the cellar
```
Comment (mobjohnson) [inline class03/abera/wine.rb]: I would recommend including the array with the wine hashes at the top of this file. Even if you are running your code by copying the separate lines into the terminal, having that array at the top of this file gives you the context of your program without having to look at the markdown file.

File: class03/abera/wine.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#4 return an array listing the unique types of wine
+ 
+unique_wines = wine_cellar.map do |types|
+types[:type]
+ 
+puts unique_wines.uniq
+ 
+#5 return an array with the al the wines that have 2-words labels
+wine_with_two_words = wine_cellar.select do |names|
+names[:type.index ("  ")
```
Comment (mobjohnson) [inline class03/abera/wine.rb]: This is missing a closing ]

**Comment by mobjohnson** [comment]
Good Job! I added some comments to your code, but you need not resubmit.

**Comment by AberaKebede** [comment]
Will do, 
Thanks

**Comment by AberaKebede** [comment]
Good evening Martin,

I was out of town due to work and I return last night.  Could you please
tell me what home work we supposed to complete.

Thanks,
Abera

**Comment by mobjohnson** [comment]
gregs_list

do git pull upstream master and it will be in class08. the lectures
material is in the INSTRUCTOR folder

*Martin Johnson*
Full-Stack Web Developer
mobjohnson.com

**Pull request merged**